test/unit/alphabet/cigar/cigar_test.cpp

```diff
@@ -56,3 +56,16 @@ TEST(cigar, assign_string)
     EXPECT_EQ(uint32_t{223}, seqan3::to_rank(get<0>(c1)));
     EXPECT_EQ('M',           get<1>(c1).to_char());
 }
+
+TEST(cigar, constexpr_char_literal)
+{
+    using seqan3::operator""_cigar_operation;
+    seqan3::cigar::operation op = 'M'_cigar_operation;
+    switch (op.to_rank())
+    {
+        case 'M'_cigar_operation.to_rank():
+            break;
+        default:
+            ;
+    }
```

Comment:
After the change from @eseiler the whole switch statement doesn't add anything. 

```suggestion
    EXPECT_EQ(op.to_rank(), 1u);
```